.dockerdev/Dockerfile

```diff
@@ -0,0 +1,56 @@
+ARG RUBY_VERSION
+FROM ruby:$RUBY_VERSION-slim-buster
```

Comment:
Just to share a thought here: Debian is great, but I like more to create my containers using an Alpine image (ex. `ruby:2.7.2-alpine`) to have the bare minimum OS possible; the drawback is that you'll probably need to install some extra package and to switch from bash to sh where needed - so also staying on Debian is a good choice anyway if you prefer it :)

Reply:
If it's ok I'd prefer to keep Debian. I know that with alpine it would be lighter, but I think that for most people Debian is a more familiar system (I included ;)). When the time comes to do some dirty work in the container I appreciate it!